src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -339,6 +339,15 @@ public int getLevel()
         return level;
     }
 
+    public void onInventoryChanged()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)